tests/generator/test_rom.py

```diff
@@ -1,7 +1,7 @@
 from clvm_tools import binutils
 from clvm_tools.clvmc import compile_clvm_text
 
-from chia.full_node.generator import run_generator
```

Comment:
these calls are deliberately changed from the mempool-mode to the non-mempool mode.